components/script/timers.rs

```diff
@@ -228,18 +228,18 @@ impl OneshotTimers {
     }
 
     pub fn suspend(&self) {
-        assert!(self.suspended_since.get().is_none());
+        // Suspend is idempotent: do nothing if the timers are already suspended.
```

Comment:
While I think making freeze/thaw/suspend idempotent isn't a bad idea, I think it can let some issues fall through the cracks. For example #13716 is caused by a some Pipelines not being frozen when navigated from. We would have no idea that this was happening if suspend was idempotent. At a minimum, it might be nice to add a `warn!` when returning early here and in `resume`